handbook/company/communications.md

```diff
@@ -66,6 +66,13 @@ Fleet uses advertising to spread awareness through a broader audience and foster
 ### Events
 It's important for Fleet to engage at events. This provides an opportunity to directly engage with potential users and contributors, build relationships, gather feedback, and create a stronger sense of community and trust.
 
+#### Event lead follow-up
```

Comment:
Good edition! Who is the DRI for making sure this SLA is followed? Demand or JD?

Reply:
Make me the DRI on follow up